use LOG_CONS for syslog

We start vswitch deamon early in runlevel S when syslog service is not
available. So we log to console if syslog fails.

--

Index: new/lib/vlog.c
===================================================================
--- new.orig/lib/vlog.c	2014-08-14 22:28:37.000000000 +0200
+++ new/lib/vlog.c	2014-09-18 08:12:27.000000000 +0200
@@ -613,7 +613,7 @@
          * a pointer to the private copy to suppress memory leak warnings in
          * case openlog() does make its own copy.) */
         program_name_copy = program_name ? xstrdup(program_name) : NULL;
-        openlog(program_name_copy, LOG_NDELAY, LOG_DAEMON);
+        openlog(program_name_copy, LOG_NDELAY|LOG_CONS, LOG_DAEMON);
         ovsthread_once_done(&once);
 
         /* Now do anything that we want to happen only once but doesn't have to
